pkg/jobs/worker.go

```diff
@@ -15,6 +15,10 @@ var (
 	defaultMaxExecTime  = 60 * time.Second
 	defaultRetryDelay   = 60 * time.Millisecond
 	defaultTimeout      = 10 * time.Second
+
+	maxMaxExecCount = 5
+	maxMaxExecTime  = 5 * time.Minute
+	maxTimeout      = 1 * time.Minute
```

Comment:
The MaxExecTime is the maximum total amount of time the worker can take working on the same job (retries and retries delay counted).

I can increase the defaults. One other option would be to have these "global" defaults kept quite low, and the ability on the worker config side (hard-coded) to have greater defaults so that the worker configuration takes precedence over the "global" if informed.